variables.pkr.hcl

```diff
@@ -64,13 +64,11 @@ variable "containerd_version" {
 variable "docker_version_al2023" {
   type        = string
   description = "Docker version to build AL2023 AMI with."
-  default     = "20.10.17"
```

Comment:
Do we not need to replace this with the new Docker version?

Reply:
It's now moved to [release.auto.pkrvars.hcl](https://github.com/aws/amazon-ecs-ami/pull/131/files#diff-2aef79116355f93c25bd24d89d1712a1d299da53b1de628e6eb041b76d2814c5), together with containerd version. I feel like they should rather be release variables than "default"s.